### PR TITLE
Fix oreg_url to work for our CI tests

### PR DESCRIPTION
**Pull request by kargakis**
@mjudeikis ptal

Fixes https://github.com/openshift/openshift-ansible/issues/9205

### CODE REVIEW

**Comment by kargakis** [comment]
Unfortunately the bot is dead, @CecileRobertMichon @jackfrancis mind merging this?

**Comment by kargakis** [comment]
> Unfortunately the bot is dead, @CecileRobertMichon @jackfrancis mind merging this?

Hold on, I think there is something more missing.

**Comment by kargakis** [comment]
This should be ready now but it's unclear why go-bindata does not cooperate. I have updated my local binary and running `make validate-generated` locally passes for me...

**Comment by jim-minter** [comment]
@kargakis you have a different version of go-bindata to the CI system?

File: parts/openshift/release-3.9/openshiftmasterscript.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ sed -i -e "s#--loglevel=2#--loglevel=4#" /etc/sysconfig/${SERVICE_TYPE}-master-c
 
 rm -rf /etc/etcd/* /etc/origin/master/* /etc/origin/node/*
 
-MASTER_OREG_URL="$IMAGE_PREFIX/$IMAGE_TYPE"
+MASTER_OREG_URL="$IMAGE_PREFIX/$IMAGE_TYPE-${component}:${version}"
```
Comment (jim-minter) [inline parts/openshift/release-3.9/openshiftmasterscript.sh]: @kargakis is it correct that `${component}:${version}` is not shell /single/ quoted?  Can this work as is?

Comment (kargakis) [reply parts/openshift/release-3.9/openshiftmasterscript.sh]: dollar signs need to be escaped, fixed!

**Comment by kargakis** [comment]
> @kargakis you have a different version of go-bindata to the CI system?

Both CircleCI and I are fetching go-bindata on the spot (`go get github.com/go-bindata/go-bindata/...`)

**Comment by CecileRobertMichon** [comment]
@kargakis let me know when this is ready, I'll run the tests and merge it

**Comment by kargakis** [comment]
this is ready now

**Comment by kargakis** [comment]
For some reason, with this fix ansible breaks the image config format for the router and the registry
```
failed: [localhost] (item={u'name': u'router', u'certificate': {u'keyfile': u'/etc/origin//master/openshift-router.key', u'certfile': u'/etc/origin//master/openshift-router.crt', u'cafile': u'/etc/origin//master/ca.crt'}, u'replicas': u'1', u'serviceaccount': u'router', u'namespace': u'default', u'selector': u'region=infra', u'edits': [{u'action': u'put', u'key': u'spec.strategy.rollingParams.intervalSeconds', u'value': 1}, {u'action': u'put', u'key': u'spec.strategy.rollingParams.updatePeriodSeconds', u'value': 1}, {u'action': u'put', u'key': u'spec.strategy.activeDeadlineSeconds', u'value': 21600}], u'images': u'openshift/origin-${component}:${version}-${component}:${version}', u'stats_port': 1936, u'ports': [u'80:80', u'443:443']}) => {"changed": false, "item": {"certificate": {"cafile": "/etc/origin//master/ca.crt", "certfile": "/etc/origin//master/openshift-router.crt", "keyfile": "/etc/origin//master/openshift-router.key"}, "edits": [{"action": "put", "key": "spec.strategy.rollingParams.intervalSeconds", "value": 1}, {"action": "put", "key": "spec.strategy.rollingParams.updatePeriodSeconds", "value": 1}, {"action": "put", "key": "spec.strategy.activeDeadlineSeconds", "value": 21600}], "images": "openshift/origin-${component}:${version}-${component}:${version}", "name": "router", "namespace": "default", "ports": ["80:80", "443:443"], "replicas": "1", "selector": "region=infra", "serviceaccount": "router", "stats_port": 1936}, "msg": {"cmd": "/usr/bin/oc create -f /tmp/router-9uW9yF -n default", "results": {}, "returncode": 1, "stderr": "Error from server: error when creating \"/tmp/router-9uW9yF\": timeout\n", "stdout": ""}}
```
Error is unrelated but you can see that the `images` field is wrong (`openshift/origin-${component}:${version}-${component}:${version}`).

**Comment by kargakis** [comment]
> For some reason, with this fix ansible breaks the image config format for the router and the registry

OK, I think I figured this out (once again).

**Comment by jim-minter** [comment]
lgtm - @CecileRobertMichon or @jackfrancis please merge

**Pull request merged**